src/main/java/frc/robot/commands/ShootingPosCommands/AmpPosition.java

```diff
@@ -30,6 +30,12 @@ public boolean isFinished(){
         return true;
     }
 
+    @Override
+    public void end(boolean i){
+        pivot.pivotBasePosition();
+        flywheel.rampDown();
+    }
+
     
```

Comment:
Remember that this will run when the command ends. Since `isFinished()` always returns true, this will immediately execute which will essentially make this command not do anything.

If you update your end condition for the command, this should work as intended.

Reply:
Ok, cool. I will look back and try to update it.